pkg/terra/client/gas_price_estimator.go

```diff
@@ -91,20 +90,24 @@ type prices struct {
 }
 
 func (gpe *FCDGasPriceEstimator) request() (map[string]sdk.DecCoin, error) {
-	req, _ := http.NewRequest(http.MethodGet, gpe.fcdURL.String(), nil)
+	fcdURL := gpe.cfg.FCDURL()
```

Comment:
If we don't have a global default fallback for this, then this should probably sanity check if it's empty and return a clear misconfiguration error.

Reply:
can we do the sanity check at chain add time? it's be cleaner if the config interface just always returns non-empty values